src/Http/Controllers/Campaigns/CampaignsController.php

```diff
@@ -61,7 +61,23 @@ public function __construct(
     public function index(): ViewContract
     {
         $workspaceId = Sendportal::currentWorkspaceId();
-        $campaigns = $this->campaigns->paginate($workspaceId, 'created_atDesc', ['status']);
+        $params = ['draft' => true];
+        $campaigns = $this->campaigns->paginate($workspaceId, 'created_atDesc', ['status'], 25, $params);
+
+        return view('sendportal::campaigns.index', [
+            'campaigns' => $campaigns,
+            'campaignStats' => $this->campaignStatisticsService->getForPaginator($campaigns, $workspaceId),
+        ]);
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function sent(): ViewContract
```

Comment:
I think that, if we're going to go down the route of splitting draft and sent as domain concepts, we should move this to its own controller as an index method.

Alternatively, we should just make draft/sent be query parameters that are sent to the single campaign index, as that's really the only difference at the moment.

Reply:
This was basically replicated from [`MessagesController`](https://github.com/mettle/sendportal-core/blob/efac212b87d10239beb97a0f91c90044ef4d8e3d/src/Http/Controllers/MessagesController.php#L76) to keep it consistent.
I think I'd pick the params only because a controller feels like overkill but happy to go with whichever people feel is most appropriate.